tasks/portfolio_intake.py

```
@@ -0,0 +1,154 @@
+
+log = logging.getLogger("qwen-proxy")
+
+# Cap: max tasks per refresh cycle
+_PORTFOLIO_MATERIALIZE_MAX = int(os.environ.get("PORTFOLIO_MATERIALIZE_MAX", "3"))
+
+# Flag: default ON (read from the canonical config module)
+def _portfolio_materialize_enabled() -> bool:
+    try:
+        from packages.config import settings
+        return settings.portfolio_materialize_enabled == "true"
+    except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)